native/explorer/src/series.rs

```diff
@@ -661,6 +661,29 @@ pub fn s_n_distinct(data: ExSeries) -> Result<usize, ExplorerError> {
     Ok(s.n_unique()?)
 }
 
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn s_pow(data: ExSeries, other: ExSeries) -> Result<ExSeries, ExplorerError> {
+    let s = &data.resource.0;
+    let s1 = &other.resource.0;
+
+    let iter1 = s.i64()?.into_iter();
+    match s1.strict_cast(&DataType::UInt32) {
+        Ok(s1) => {
+            let iter2 = s1.u32()?.into_iter();
+
+            let s = iter1
+                .zip(iter2)
+                .map(|(v1, v2)| v1.unwrap().pow(v2.unwrap()))
```

Comment:
Is it possible for one of values to be nil/null? Could you add a test for that if so?